frontend/src/modules/samples/actions.js

```diff
@@ -51,6 +52,22 @@ export const list = (options) => async (dispatch, getState) => {
     ));
 };
 
+export const listFilter = ({ offset = 0, limit = DEFAULT_PAGINATION_LIMIT, filters = {}, sortBy, filterKey }) => async (dispatch, getState) => {
```

Comment:
Not sure why you need filterKey in here ... You can test empty filters otherwise without having a key.
Also why is there no abort param for this list action ? i don't think its used a lot but i think it is cleaner to have it for every list action.

Reply:
When the component is mounted, for a few seconds, the filtering ID is undefined. Thus, checking if the ID is not undefined is necessary to filter properly.

Reply:
This is an element related to the component and it's loading state ... Shouldn't this be part of isFetching or isLoading ? I do not see this as normal to include in a function a value that is not used in it just for the sake of testing it. if its undefined when you call listFilter ... you can avoid calling listFilter since you'll just exit anyway ...

Reply:
Yes, that could work better actually